app/models/rent.rb

```diff
@@ -0,0 +1,6 @@
+class Rent < ApplicationRecord
+  belongs_to :user
+  belongs_to :book
+  validates :user, presence: true
+  validates :book, presence: true
```

Comment:
Consider adding a validation to the :from and :to fields

Reply:
I just did it